src/com/esotericsoftware/kryo/serializers/DefaultSerializers.java

```diff
@@ -852,6 +853,27 @@ private PriorityQueue createPriorityQueue (Class<? extends Collection> type, int
 		}
 	}
 
+	/** Serializer for {@link ConcurrentHashMap.KeySetView}.
+	 * @author Andreas Bergander */
+	public static class KeySetViewSerializer extends CollectionSerializer<ConcurrentHashMap.KeySetView> {
+		protected void writeHeader (Kryo kryo, Output output, ConcurrentHashMap.KeySetView set) {
+			kryo.writeClassAndObject(output, set.getMappedValue());

```

Comment:
Is there a reason for writing the `mappedValue` first? If not, then please write the map first and change the order in `createKeySetView` as well.

Apart from that LGTM.

Reply:
I've swapped the order of those fields now.